openshift/server.dc.yml

```diff
@@ -119,24 +119,18 @@ objects:
                       key: sa-password
                       name: ${APP_NAME}-keycloak
                 - name: MONGO_HOST
-                  value: ${APP_NAME}-mongodb
+                  value: ${APP_NAME}-mongo-headless
                 - name: MONGO_PORT
                   value: '27017'
                 - name: MONGO_DB
-                  valueFrom:
-                    secretKeyRef:
-                      key: database
-                      name: ${APP_NAME}-mongodb
+                  value: ${APP_NAME}
```

Comment:
This should be connected to any secret. This is a public repository, this is a big security issue. or at-least connect these auth attribute with input parameter